docs/standards/tokens/LSP4-Digital-Asset-Metadata.md

```diff
@@ -69,6 +69,38 @@ The value attached to this data key represents the name of the digital asset.
 
 The value attached to this data key represents the symbol of the digital asset.
 
+### `LSP4TokenType`
+
+:::caution Warning
+
+This MUST NOT be changeable, and set only during initialization of the token contract.
+
+:::
+
+```json
+{
+  "name": "LSP4TokenType",
+  "key": "0xe0261fa95db2eb3b5439bd033cda66d56b96f92f243a8228fd87550ed7bdfdb3",
+  "keyType": "Singleton",
+  "valueType": "uint256",
+  "valueContent": "Number"
+}
+```
+
+The value attached to this data key repesents the type of token of the digital asset.
+
+:::note References
+
+More token types COULD be added later.
```

Comment:
LSP4 defines 3 default token types (see table below). However, these are not restrictive. Applications and protocols can define new custom token types starting from `3`.